doc/clone-from-volumesnapshot-source.md

```diff
@@ -0,0 +1,82 @@
+# Data Volume cloning from a VolumeSnapshot source
+
+## Introduction
+Current CDI PVC cloning methods offer [smart cloning](./smart-clone.md) which uses a 1:1 cloning method that creates a temporary snapshot of the source PVC for each intended clone.  
+Some storage systems are designed to scale better with a 1:N method where a single snapshot of the source PVC is used to create multiple clones.  
+One examples is ceph, where this would be about utilizing RBD layering.  
+A much simpler example would be storage that can take around 5-10 minutes to snapshot a 200GB volume.  
+
+### Note: Data Volume cloning from snapshot source can work together with namespace transfer and size expansion  
```

Comment:
Note is usually marked with `>` or `*` not as another headline i.e ###

Reply:
thx, done